cmd/loop/loopin.go

```diff
@@ -67,6 +67,21 @@ func loopIn(ctx *cli.Context) error {
 	}
 
 	limits := getInLimits(amt, quote)
+	external := ctx.Bool("external")
+	
+	// There could have been an error in the on-chain fee estimation if
```

Comment:
Following from my last comment, we can use `Is/As` here instead and gate the warning or exit based on that.